perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/NotificationManagerMethod.java

```diff
@@ -49,15 +50,37 @@ public PerunNotifReceiver call(ApiCaller ac, Deserializer parms) throws PerunExc
 	/*#
 	 * Saves PerunNotifReceiver to db and creates <code>id</code>.
 	 *
+	 * PerunNotifReceiver must contain: target, locale, templateId, key. Parameters target, locale, key are optional.
+	 *
 	 * @param receiver PerunNotifReceiver PerunNotifReceiver object without <code>id</code>
 	 * @return PerunNotifReceiver PerunNotifReceiver with new <code>id</code> set
 	 */
+	/*#
+	 * Saves PerunNotifReceiver to db and creates <code>id</code>.
+	 *
+	 * @param target String Defines target of receiver, usually contains function to get email or jabber number...
+	 * @param locale String Locale of receiver - determines a language, which is used for sending messages
+	 * @param templateId int Template Id to which receiver is connected
+	 * @param key String Type of the receiver - 'EMAIL_USER' or 'EMAIL_GROUP'
```

Comment:
It is weird to use word ``key`` as param name for type of receiver. Please change it to ``type`` or something like this.